salt/beacons/btmp.py

```diff
@@ -70,7 +70,7 @@ def beacon(config):
             __context__[LOC_KEY] = fp_.tell()
             pack = struct.unpack(FMT, raw)
             event = {}
-            for ind in range(len(FIELDS)):
+            for ind in enumerate(FIELDS):
```

Comment:
Could we just do
```python
for ind, fields in enumerate(FIELDS):
    event[FIELDS[ind]] = pack[ind]
    if isinstance(event[FIELDS[ind]], str):
        event[FIELDS[ind]] = event[FIELDS[ind]].strip('\x00')
```

Basically nothing changes except add the extra variable for unpacking. I could be very wrong in my thinking here, so feel free to correct me :)